ethereum-rpc/src/constructors/ethereum/index.js

```diff
@@ -3,7 +3,8 @@ const path = require('path')
 const checksum = require('ethereum-checksum-address')
 // const decoder = require('abi-decoder')
 
-const web3 = require('@/constructors/web3')
+const web3Connection = require('@/constructors/web3')
+const {web3,bscWeb3} = web3Connection;
```

Comment:
Let's export just web3, and in the case of binance smart chain, export a different web3 instance

Reply:
ok , let I will take another web3 instance for bsc